### PR TITLE
Pgiraud v4 layout

### PR DESCRIPTION
**Pull request by pgiraud**

### CODE REVIEW

**Comment by elemoine** [comment]
You probably want to remove `data/thinkhazard.xml`.

File: thinkhazard/__init__.py [hunk 1]
```diff
@@ -18,6 +19,7 @@ def main(global_config, **settings):
     config = Configurator(settings=settings)
 
     config.include('pyramid_jinja2')
+    config.include('papyrus.includeme')
```
Comment (elemoine) [inline thinkhazard/__init__.py]: Use `config.include('papyrus')` for consistency with the way we include `pyramid_jinja2`.

File: thinkhazard/static/js/report.js [hunk 2]
```diff
@@ -1,138 +1,92 @@
...
+    controls: [],
+    layers: [
+      new ol.layer.Tile({
+        source: new ol.source.Stamen({
+          layer: 'watercolor'
+        }),
+        opacity: 0.5
+      })
+    ],
+    view: new ol.View({ })
```
Comment (elemoine) [inline thinkhazard/static/js/report.js]: An `ol.View` object will be created for you if you don't provide one.

File: thinkhazard/static/js/report.js [hunk 3]
```diff
@@ -1,138 +1,92 @@
...
+        stroke: new ol.style.Stroke({
+          color: 'rgba(255, 255, 255, 0.6)',
+          width: 5
+        }),
+        fill: new ol.style.Fill({
+          color: 'black'
+        })
+      })
+    })];
+  };
```
Comment (elemoine) [inline thinkhazard/static/js/report.js]: For performance reasons we should create the styles array and the static styles outside the style function. But we can do this with a separate PR, when we add the choropleths.

File: thinkhazard/static/js/report.js [hunk 4]
```diff
@@ -1,138 +1,92 @@
...
+    style: styleFn,
+    source: new ol.source.Vector({
+      url: app.mapUrl,
+      format: new ol.format.GeoJSON({
+        defaultDataProjection: 'EPSG:3857'
+      })
+    })
   });
+  map.addLayer(vector);
+  window.source = vector.getSource();
```
Comment (elemoine) [inline thinkhazard/static/js/report.js]: You want to remove this.

File: thinkhazard/templates/report.jinja2 [hunk 5]
```diff
@@ -166,11 +165,29 @@
...
-      app.hazardType = '{{hazard.hazardtype.mnemonic}}';
-    {% endif %}
-  </script>
+{% if debug %}
+    <script src="{{('%s/openlayers/dist/ol-debug.js' % node_modules)|static_url}}"></script>
+{% else %}
+    <script src="{{('%s/openlayers/dist/ol.js' % node_modules)|static_url}}"></script>
+{% endif %}
+    <script>
+      {% if hazard %}
```
Comment (elemoine) [inline thinkhazard/templates/report.jinja2]: I'd suggest using `{%- if hazard -%}` everywhere to avoid blank lines. See http://jinja.pocoo.org/docs/dev/templates/#whitespace-control.

File: thinkhazard/templates/report.jinja2 [hunk 6]
```diff
@@ -166,11 +165,29 @@
...
+      {% endif %}
+
+      {%- if division %}
+      app.divisionCode = {{division.code}};
+      {%- else %}
+      app.divisionCode = undefined;
+      {%- endif %}
+
+      app.divisionBounds = {{bounds}};
+      {% if hazard %}
```
Comment (elemoine) [inline thinkhazard/templates/report.jinja2]: `{%- if hazard -%}`

File: thinkhazard/templates/base.jinja2 [hunk 7]
```diff
@@ -30,13 +30,6 @@
       var app = {};
       app.administrativedivisionUrl = '{{'administrativedivision'|route_url}}';
       app.reportpageUrl = '{{'report_overview'|route_url(divisioncode='divisioncode')}}';
```
Comment (elemoine) [inline thinkhazard/templates/base.jinja2]: What happens if you use the following?

```
app.reportpageUrl = '{{'report_overview'|route_url(divisioncode='{{divisioncode}}')}}';
```

**Comment by elemoine** [comment]
A couple minor comments. Otherwise looks good to me. Please merge when addressed.

File: thinkhazard/templates/report.jinja2 (hunk 5 above)
Comment (pgiraud) [inline thinkhazard/templates/report.jinja2]: I will use jinja2 trim_blocks option.

**Comment by pgiraud** [comment]
Comments addressed, merging.
